season/week_two_challenges/more_methods.md

```diff
@@ -0,0 +1,117 @@
+# More Method Madness
+
+## Method of Thrones
+
+Create a method called `character_house`.
+* This method should take two string arguments: `first_name` and `house_name`.
+* When executed, the method should return "`first_name`, house of `house_name`".
+
+Example:
+```ruby
+character_house("Tyrion", "Lannister")
+# output => "Tyrion, house of Lannister."
+```
+```ruby
+def character_house(first_name,house_name)
+ puts "#{first_name}, house of  #{house_name}"
+end
+character_house("Tyrion", "Lannister")
+```
+
+## Triangulate It
+
+Create a method called `triangle_checker`.
+* It should take three numbers as arguments.
+* Each of those arguments represents an angle.
+* If those three numbers add up to 180, then return `true`.
+* If those three numbers do not add up to 180, then return `false`.
+
+```ruby
+def triangle_checker(no1,no2,no3)
+  if no1 + no2 + no3 == 180 
+    return true 
+  else
+    return false
+  end
+end
+triangle_checker(60,60,60)
+```
+
+## Fizz Buzz
+
+Everyone's favorite! Create a method called `fizz_buzz` that takes a number (`max_number`) as an argument.
+
+When `fizz_buzz(max_number)` is executed, it should do the following for every number from 0 *up to* the `max_number`:
+* If a number is divisible by 3, it should log the word "Fizz" to the console.
+* If a number is divisible by 5, it should log "Buzz".
+* If a number is divisible by both 3 and 5, it should log "FizzBuzz".
+* Otherwise, it should just log the number.
+```ruby
+def fizz_buzz(number)
+  for i in 0..(number) do 
+    if i % 5 == 0 and i % 3 == 0
+      puts "FizzBuzz"    
+    elsif i % 3 == 0 
+      puts "Fizz"
+    elsif i % 5 == 0 
+      puts "Buzz"
+    else 
+      puts number
+    end
+  end
+end
+fizz_buzz(5)
+```
```

Comment:
Hmm, something is fishy here. Here's my output:
```
fizz_buzz(15)
FizzBuzz
15
15
Fizz
15
Buzz
Fizz
15
15
Fizz
Buzz
15
Fizz
15
15
FizzBuzz
```

Reply:
what.....?